### ISSUE TITLE
Debian patches to onesixtyone

### ISSUE THREAD

**Issue by charles2910** [trailofbits/onesixtyone#24]
Hi, there are a couple patches Debian is applying to onesixtyone and it would be nice to see them incorporated upstream.
They are:

```diff
Description: Add missing flags
Author: Sophie Brun <sophie@offensive-security.com>
Last-Update: 2019-04-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 onesixtyone: onesixtyone.c
-	gcc -o onesixtyone onesixtyone.c
+	gcc $(CFLAGS) $(LDFLAGS) $(CPPFLAGS) -o onesixtyone onesixtyone.c
 
 solaris: onesixtyone.c
 	cc -o onesixtyone onesixtyone.c -lsocket -lnsl
```

and

```diff
Description: don't hard codes the build architecture compiler
 onesixtyone fails to cross build from source, because the upstream
 Makefile hard codes the build architecture compiler "gcc". Making it
 substitutable is sufficient for making onesixtyone cross buildable.
Author: Helmut Grohne <helmut@subdivi.de>
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=930902
Last-Update: 2019-07-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 onesixtyone: onesixtyone.c
-	gcc $(CFLAGS) $(LDFLAGS) $(CPPFLAGS) -o onesixtyone onesixtyone.c
+	$(CC) $(CFLAGS) $(LDFLAGS) $(CPPFLAGS) -o onesixtyone onesixtyone.c
 
 solaris: onesixtyone.c
 	cc -o onesixtyone onesixtyone.c -lsocket -lnsl
```

If you'd like to see them upstreamed, I could open a PR.

Cheers!

**Comment by charles2910** [comment]
There is also a [manpage](https://salsa.debian.org/pkg-security-team/onesixtyone/-/blob/debian/master/debian/onesixtyone.1) based on the helper text printed by onesixtyone.

**Comment by woodruffw** [comment]
Hi @charles2910! Yes, please do open a PR with those changes, thanks!

**Comment by charles2910** [comment]
Thanks for merging. It'd be really cool if you released a new version - 0.3.4 I think :-)

**Comment by woodruffw** [comment]
Sure, I can do that tomorrow. Sent from mobile. Please excuse my brevity.On Dec 27, 2022, at 7:42 PM, Carlos Henrique Lima Melara ***@***.***> wrote:﻿
Thanks for merging. It'd be really cool if you released a new version - 0.3.4 I think :-)

**Comment by woodruffw** [comment]
Cut with 0.3.4. Thanks @charles2910!
